docs/pages/x/api/date-pickers/desktop-next-time-picker.json

```diff
@@ -92,10 +92,7 @@
   },
   "name": "DesktopNextTimePicker",
   "styles": { "classes": [], "globalClasses": {}, "name": "MuiDesktopNextTimePicker" },
```

Comment:
We need to improve the `parseTest` method on the core repo to handle the new test file names
I propose to do it in a follow up PR, as in any case it was already broken for most components

Reply:
The information lost in this file are parsed from the `describeConformance` test
And `parseTest` look from a test file in the same folder as the component file

Reply:
#7294 is continuing this discussion.